rlclientlib/schema/v2/ModelUpdateEvent.fbs

```diff
@@ -0,0 +1,10 @@
+﻿namespace reinforcement_learning.messages.flatbuff.v2;
+
+table ModelUpdate {
+    // VW ModelDelta
+    delta: [ubyte];
```

Comment:
Is this not just a collection of (hash, float) values?

Reply:
No a VW ModelDelta is pretty much a full VW model. This is to adhere to the model merging operations defined in VW.

Reply:
To be specific, my concern is around versioning.  If this is an opaque payload that changes over time, we would then have to worry about cases where the sender and reciever might be out of sync. Where as being specific about schema means we can simply send a different message that's either handled or not.

Reply:
Ah ok.  makes sense.  Thanks

Reply:
This work is still experimental, so I have we can update to consume the schematized VW model so we won't need to worry.

For now though it follows the current "status quo" of VW models, so its version is embedded within it and as long as the format doesn't change should be loadable.